mne/tests/test_event.py

```diff
@@ -372,4 +377,29 @@ def test_define_events():
     assert_array_equal(true_lag_fill, lag_fill)
     assert_array_equal(true_lag_nofill, lag_nofill)
 
+
+def test_elekta_averager():
+    """Test averaging according to Elekta DACQ parameters"""
+    raw = io.read_raw_fif(fname_raw_elekta, preload=True)
+    eav = Elekta_averager(raw.info['acq_pars'])
+    for cat in eav.categories:
+        # XXX datasets match only when baseline is applied to both,
+        # not sure where relative dc shift comes from
+        eps = cat.epochs(raw, baseline=(-.05, 0))
```

Comment:
at least

epochs = cat.get_epochs(raw, baseline=(-.05, 0))

but why not

epochs = eav.get_epochs(raw, category=cat, baseline=(-.05, 0))

?

this would avoid another object